airmozilla/starred/tests/test_starred.py

```diff
@@ -0,0 +1,198 @@
+import json
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from funfactory.urlresolvers import reverse
+from nose.tools import eq_, ok_
+
+from airmozilla.main.models import Event
+from airmozilla.starred.models import (
+    StarredEvents,
+)
+
+
+class TestStarredEvent(TestCase):
+    fixtures = ['airmozilla/manage/tests/main_testdata.json']
+# test that a starred event can be synced
+# test that an unstarred event can be synced
+# test that unchanged events remain that way
+# test that unknown event id does the right thing (whatever that is)
+# test that the proper list is returned from a get
+# test that the proper list of starred events is displayed
+# add star, sync 
+# remove star, add different star, sync
+
+# test what happens when a user isn't logged in
+
+
+    def setUp(self):
+        super(TestStarredEvent, self).setUp()
+
+        # create the url
+        self.url = reverse('starred:sync_starred_events')
+        # so, let's sign in
+        User.objects.create_user('lisa', password='secret')
+        assert self.client.login(username='lisa', password='secret')
```

Comment:
Once you've changed the test case base class you'll be able to use `self._login()` to get a user to be created and logged in to the `self.client`.